src/bitcoin-cli.cpp

```diff
@@ -747,9 +743,8 @@ static UniValue CallRPC(BaseRequestHandler* rh, const std::string& strMethod, co
     raii_evhttp_request req = obtain_evhttp_request(http_request_done, (void*)&response);
     if (req == nullptr)
         throw std::runtime_error("create http request failed");
-#if LIBEVENT_VERSION_NUMBER >= 0x02010300
+
```

Comment:
```suggestion
}
```

nit: Maybe add `{}` while touching the line?

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/24681#issuecomment-1083053772).